bot/code_coverage_bot/artifacts.py

```diff
@@ -25,41 +30,65 @@ def __init__(self, task_ids, parent_dir="ccov-artifacts", task_name_filter="*"):
         self.task_ids = task_ids
         self.parent_dir = parent_dir
         self.task_name_filter = task_name_filter
+        self.artifacts = []
 
     def generate_path(self, platform, chunk, artifact):
         file_name = "%s_%s_%s" % (platform, chunk, os.path.basename(artifact["name"]))
         return os.path.join(self.parent_dir, file_name)
 
     def get_chunks(self, platform):
         return set(
-            f.split("_")[1]
-            for f in os.listdir(self.parent_dir)
-            if os.path.basename(f).startswith(f"{platform}_")
+            artifact.chunk
+            for artifact in self.artifacts
+            if artifact.platform == platform
         )
 
-    def get(self, platform=None, suite=None, chunk=None):
-        files = os.listdir(self.parent_dir)
+    def get_suites(self):
```

Comment:
Nit: get_suite_platform_combinations?

Reply:
Counter-nit: `get_combinations` (as it's more generic to support future filters)